spec/client_spec.rb

```diff
@@ -308,6 +308,31 @@
     end
   end
 
+  describe "#locked?" do
```

Comment:
You could use single quotes here and below for all strings that do not contain interpolation.